packages/api-server/src/convert-tx.ts

```diff
@@ -133,8 +175,9 @@ function encodePolyjuiceTransaction(tx: PolyjuiceTransaction) {
 
 async function parseRawTransactionData(
   rawTx: PolyjuiceTransaction,
-  rpc: GodwokenClient
-): Promise<L2Transaction> {
+  rpc: GodwokenClient,
+  polyjuiceRawTx: HexString
+): Promise<[L2Transaction, [string, string] | undefined]> {
```

Comment:
I suggest that splits [the cache logic](https://github.com/nervosnetwork/godwoken-web3/pull/430/files#diff-9b0dee551144978abbf32f56e4753f613c673ca1fce488d1d3f754556500f399R229-R240) out from this function. `parseRawTransactionData` should only parse transaction data.

Reply:
This function aims to convert `PolyjuiceTransaction` to `L2Transaction`, and signature should recovered here and get `from_id`, so if auto create account tx, should set `from_id = 0` instead of throw error.
And cache generated here because of `fromAddress` already recovered here.